### DISCUSSION TITLE
what is this issue about  media cannot not load on https

### DISCUSSION THREAD

**Discussion by samchan952** [thedevdojo/voyager#5811] [Q&A]
![image](https://github.com/thedevdojo/voyager/assets/83875198/cb165298-499f-4f7a-8dad-8a9121072022)

Mixed Content: The page at 'https://xxx/admin/media' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://xxx/admin/media/files'. This request has been blocked; the content must be served over HTTPS.

**Comment by gcorp-llc** [comment]
check .env file and set https url to APP_URL parameter and run php artisan optimize:clear 
and check .htassess file for auto redirect http to https and host panel

**Comment by youseffatouraee** [comment]
If you remove env('APP_URL'). from the public disk configuration, then it will render a domain-relative url, which will always use the current domain and protocol.
As an aside, if you need a fully-qualified URL, you could wrap the call to Voyager::image('...') with asset(), so it would be
`asset(Voyager::image('...'))`
This will return the current protocol, domain, and correct path to that image.](https://voyager-docs.devdojo.com/troubleshooting/using-https)
